### DISCUSSION TITLE
I would like to add Thai as a definition language to english book.

### DISCUSSION THREAD

**Discussion by Snickth** [xxyzz/WordDumb#271] [Q&A]
hello guys, I am a part time translator, and I really want worddumb to support thai, I guess the problem is the definition.
So im thinking of making the translation myself but I need help for the tools and how to do it.
I studied It related so im pretty familiar to software.
 Please feel free to recommend.

Thank you in advance!

**Comment by xxyzz** [comment]
Could use the translation data from the English Wiktionary, English to Hebrew data was added in this way. Should be easy to implement, I could take a look next week.

**Comment by xxyzz** [comment]
English to Thai files are added, please try the test version: https://github.com/xxyzz/WordDumb/actions/runs/11704233163

**Comment by Snickth** [comment]
Hello, Sorry , I just logged back into the forum and found your reply,  is the file expired to download? can you please reupload it again? I will make sure I check back on this everyday for the whole week.

**Reply by xxyzz** [reply]
You can install the latest version, these changes were already included in recent releases.

**Reply by Snickth** [reply]
Wow , it worked really amazing! thanks so much
